pkg/operator/operator.go

```diff
@@ -18,7 +18,12 @@ func ValidateRBAC(cs *clients.Clients, rnames config.ResourceNames) {
 	EnsureTektonConfigStatusInstalled(cs.TektonConfig(), rnames)
 	//Verify `pipelineSa` is created in any namespace
 	AssertServiceAccount(cs, store.Namespace(), "pipeline")
-	//Verify roleBindings are created in any namespace
+	//Verify clusterrole
+	AssertClusterRole(cs, "pipelines-scc-clusterrole")
+	//Verify configmaps is created in any namespace
+	AssertConfigMap(cs, store.Namespace(), "config-service-cabundle")
```

Comment:
In the future, if we want to verify a different configmap that is not part of another test case, the step `Validate RBAC` cannot be used right? Isn't it better to have a different step for verifying the configmaps? WDYT?

Reply:
makes sense I would rather rename it to `validateOperatorRBAC` WDYS??

Reply:
+1, please change the step as well in test case

Reply:
I guess it makes sense to keep name as expected as it is under package operator.